src/lib.rs

```diff
@@ -29,6 +29,64 @@ fn jsonschema_is_valid(schema: Json) -> bool {
     }
 }
 
+#[pg_extern(immutable, strict)]
+fn validate_json_schema(schema: Json, instance: Json) -> bool {
```

Comment:
From the linked issue I thought the goal was to produce functions that return an array of strings that described and errors that are present.

These two functions have identical signatures and behavior to `json_matches_schema` and `jsonb_matches_schema` but have raise notices that list the errors.

If that is the goal, why not update the existing two functions? e.g. is it a performance problem? if so, what is the penalty?

If it is something like performance, an optional argument to the existing functions like `on_error_raise` that takes one of `nothing` (default), `notice`, or `exception` would be preferred to adding new functions

Reply:
Yes, there is performance problem.
From the doc, the `jsonschema::is_valid` is much faster than `validate` function.

Anyhow, I removed the new functions, and updated the existing functions as recommended.
Please re-check it. @olirice